### ISSUE TITLE
Admin dashboard not loading properly

### ISSUE THREAD

**Issue by Ulrar** [jellyfin/jellyfin-web#2405]
**Describe the bug**
While trying to find a solution to [another issue](https://www.reddit.com/r/jellyfin/comments/lhtn5b/livetv_hardware_transcoding_issue/) I've upgraded to linuxserver/jellyfin:nightly and now the admin dashboard does not load properly, I get the Scan All library and shutdown button and some empty boxes, but no information loads in them.

The rest of the admin area still works perfectly fine so I can still create users and change settings, just no overview.
Same behavior in a browser and in the Android app.

**System (please complete the following information):**
 - OS: debian buster
 - Virtualization: docker (linuxserver/jellyfin:nightly)
 - Clients: Android
 - Browser: Latest Windows firefox
 - Jellyfin Version: linuxserver/jellyfin:nightly@sha256:ab6d5079c5ebf9d9e0450963378aa302048d7d0fe53def4ebc8d3138975aaa42
 - Playback: No issues
 - Installed Plugins: Kodi Sync Queue
 - Reverse Proxy: HAProxy
 - Base URL: /
 - Networking: Macvlan (so dedicated IP)
 - Storage: local

I realize you have your own docker image but unfortunately I wasn't able to get hardware transcoding working with yours, as I have a pretty recent intel that requires the iHD driver. Linuxserver's container on the other hand just worked out of the box, so I went with that. Hopefully that's not related to the issue here.

**To Reproduce**
Go to the admin dashboard. I have to click the link twice in the side menu before it actually switches to it, but it never loads properly.

**Expected behavior**
Get the dashboard to load properly

**Logs**
That's the error in firefox's console : 
```
Uncaught TypeError: g is undefined
    D https://myjellyfin.tld/web/dashboard-dashboard.bundle.js:1
    P https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    promise callback*value https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    promise callback*value https://myjellyfin.tld/web/main.bundle.js:2
    s https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    promise callback*value https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    promise callback*value https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    middleware https://myjellyfin.tld/web/main.bundle.js:2
    i https://myjellyfin.tld/web/main.bundle.js:2
    e https://myjellyfin.tld/web/main.bundle.js:2
    dispatch https://myjellyfin.tld/web/main.bundle.js:2
    show https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    value https://myjellyfin.tld/web/main.bundle.js:2
    u https://myjellyfin.tld/web/main.bundle.js:2
```

**Screenshots**
![image](https://user-images.githubusercontent.com/873423/107853485-520b4580-6e0e-11eb-8a12-ae21247c2bf3.png)

**Additional context**
Works fine on stable, but unfortunately I didn't make a backup of my DB (I know, I know) so I can't downgrade back.

Thanks

**Comment by Ulrar** [comment]
As an update I've downgraded to the nightly from the 2021/01/22 and the dashboard is working fine with that one after a cache clear. Not sure what broke the dashboard but it must be a change between the 22snd and now.

Thanks

**Comment by thornbill** [comment]
I’ve been seeing similar errors on unstable but haven’t been able to replicate the errors with a local build of the web client.

**Comment by MrLemur** [comment]
I have just logged the same issue on the main jellyfin repo: https://github.com/jellyfin/jellyfin/issues/5232

**Comment by MrLemur** [comment]
Based on the linuxserver nightly images, something changed between `nightly-version-20210208.21-unstable` and 
`nightly-version-20210208.87-unstable` that caused the issue. I'm currently combing through the commits now.

**Comment by MrLemur** [comment]
I had a hunch the issue is `date-fns` being upgraded a few days ago. PR https://github.com/jellyfin/jellyfin-web/pull/2394 from `2.16.1` to `2.17.0` seems to have introduced this.

I pulled the latest `jellyfin-web` code and ran `yarn upgrade date-fns@2.16.1` to test and I confirmed dashboard and activity are showing as expected. When I install the latest `2.17.0` version, I run into the original issue.

In summary, downgrading `date-fns` to `2.16.1` fixes the issue.

**Comment by MrLemur** [comment]
I have also looked at the module exports expected, and adding `sourceType: "unambiguous"` to `babel.config.js` also fixes the issue with date-fns `2.17.0`

**Comment by richardbrodie** [comment]
It may or may not be the same issue, but I also upgraded to the latest linuxserver nightly and can't get access to the dashboard at all, the console shows a slightly different error though

```
Uncaught (in promise) ReferenceError: exports is not defined
    36794 https://jellyfin.calcifer.se/web/7084.bundle.js:1
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    52092 https://jellyfin.calcifer.se/web/7084.bundle.js:1
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    33257 https://jellyfin.calcifer.se/web/7084.bundle.js:1
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    9785 https://jellyfin.calcifer.se/web/dashboard-dashboard.bundle.js:1
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    9899 https://jellyfin.calcifer.se/web/dashboard-dashboard.bundle.js:1
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    t https://jellyfin.calcifer.se/web/main.bundle.js:2
    o https://jellyfin.calcifer.se/web/main.bundle.js:2
    promise callback*o https://jellyfin.calcifer.se/web/main.bundle.js:2
    value https://jellyfin.calcifer.se/web/main.bundle.js:2
    value https://jellyfin.calcifer.se/web/main.bundle.js:2
    value https://jellyfin.calcifer.se/web/main.bundle.js:2
    promise callback*value https://jellyfin.calcifer.se/web/main.bundle.js:2
    value https://jellyfin.calcifer.se/web/main.bundle.js:2
    value https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    middleware https://jellyfin.calcifer.se/web/main.bundle.js:2
    i https://jellyfin.calcifer.se/web/main.bundle.js:2
    e https://jellyfin.calcifer.se/web/main.bundle.js:2
    dispatch https://jellyfin.calcifer.se/web/main.bundle.js:2
    show https://jellyfin.calcifer.se/web/main.bundle.js:2
    value https://jellyfin.calcifer.se/web/main.bundle.js:2
    value https://jellyfin.calcifer.se/web/main.bundle.js:2
    u https://jellyfin.calcifer.se/web/main.bundle.js:2
```
